package.json

```diff
@@ -9,42 +9,47 @@
     "addPrizes": "hardhat run ./scripts/addPrizes.js --network ",
     "clean": "rm -rf cache/ artifacts/",
     "compile": "hardhat compile",
-    "task": "hardhat",
-    "dashboard": "node tasks/dashboard",
+    "console": "hardhat console --network localhost",
+    "prepare-deploy": "yarn clean && yarn compile",
+    "prepack": "yarn generate",
+    "reinstall": "rm -rf node_modules/ && yarn cache clean && yarn",
     "docs": "hardhat deploy-markdown-export --name 'V4 Mainnet' --npm-package-name @pooltogether/v4-mainnet --github-base-url 'https://github.com/pooltogether/v4-mainnet/tree/master' --output-filepath mainnet.md",
-    "verify-polygon": "hardhat --network polygon etherscan-verify --api-key $POLYGONSCAN_API_KEY",
-    "verify-mainnet": "hardhat --network mainnet etherscan-verify --api-key $ETHERSCAN_API_KEY",
-    "deploy-mainnet-to-polygon": "DEPLOY=polygon hardhat deploy --network polygon",
-    "deploy-mainnet-to-ethereum": "DEPLOY=mainnet hardhat deploy --network mainnet",
-    "deploy-goerli": "DEPLOY=goerli hardhat deploy --network goerli",
     "disburse": "hardhat run ./scripts/disburse.js --network ",
     "deposit": "hardhat run ./scripts/deposit.js --network ",
     "generate": "node ./scripts/generateContractList.js",
-    "deploy-polygon": "DEPLOY=polygon hardhat deploy --network localhost",
-    "deploy-mainnet": "DEPLOY=mainnet hardhat deploy --network localhost",
-    "fork-polygon": "FORK_CHAIN_ID=$POLYGON_CHAIN_ID FORK_RPC_URL=$POLYGON_RPC_URL FORK_ENABLED=true DEPLOY=polygon hardhat node --no-deploy --network hardhat",
-    "fork-mainnet": "FORK_CHAIN_ID=$MAINNET_CHAIN_ID FORK_RPC_URL=$MAINNET_RPC_URL FORK_ENABLED=true DEPLOY=mainnet hardhat node --no-deploy --network hardhat",
-    "console": "hardhat console --network localhost",
-    "node-goerli": "DEPLOY=goerli hardhat node",
-    "prepare-deploy": "yarn clean && yarn compile",
-    "prepack": "yarn generate",
-    "reinstall": "rm -rf node_modules/ && yarn cache clean && yarn"
+    "deploy-mainnet-to-polygon": "DEPLOY=polygon hardhat deploy --network polygon",
+    "deploy-mainnet-to-ethereum": "DEPLOY=mainnet hardhat deploy --network mainnet",
+    "deploy:mainnet": "DEPLOY=mainnet hardhat deploy --network localhost",
+    "deploy:polygon": "DEPLOY=polygon hardhat deploy --network localhost",
+    "fork:mainnet": "FORK_CHAIN_ID=$MAINNET_CHAIN_ID FORK_RPC_URL=$MAINNET_RPC_URL FORK_ENABLED=true DEPLOY=mainnet hardhat node --no-deploy --network hardhat",
+    "fork:polygon": "FORK_CHAIN_ID=$POLYGON_CHAIN_ID FORK_RPC_URL=$POLYGON_RPC_URL FORK_ENABLED=true DEPLOY=polygon hardhat node --no-deploy --network hardhat",
+    "node:ethereum": "DEPLOY=ethereumMainnet hardhat node --network hardhat",
+    "node:polygon": "DEPLOY=polygonMainnet hardhat node --network hardhat",
+    "node:avalanche": "DEPLOY=avalancheMainnet hardhat node --network hardhat",
+    "upgrade:mainnet": "DEPLOY=mainnet.upgrade.v1.10.0 hardhat deploy --network mainnet",
```

Comment:
is the minor version `0.10` or `0.1`?

Reply:
You're right @aodhgan it should be `v1.1.0`.